js_modules/dagit/packages/core/src/runs/RunTableNew.tsx

```diff
@@ -326,6 +390,35 @@ const RunRow: React.FC<{
       <td>
         <RunActionsMenu run={run} onAddTag={onAddTag} />
       </td>
+      {showRunTags ? (
```

Comment:
Shouldn't use this, since `showRunTags` is already the `isOpen` value, and rendering the dialog to null will kill the hide animation.